spec/rubocop/cop/performance/use_zip_to_wrap_array_contents_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::UseZipToWrapArrayContents, :config do
+  context 'when using map with array literal' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        [1, 2, 3].map { |id| [id] }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `.zip` instead of `.map { |id| [id] }`.
+      RUBY
+    end
+
+    it 'corrects the code' do
+      new_source = autocorrect_source('[1, 2, 3].map { |id| [id] }')
```

Comment:
Use `expect_correction` and fold it into the `expect_offense` test. Same for all the others.

Reply:
I got this pattern from here: https://github.com/search?q=repo%3Arubocop%2Frubocop-performance%20new_source&type=code 
It looks common in the recent commits. Did I misapply it?

Reply:
Okay, fixed! I see it now, the other standalone tests aren't already matching the same string to expect an offense